radixdlt/babylon-wallet-android
# [ABW-2349] Remove mainnet check

## Description
[(Android)  Post-babylon cleanup: remove `isMainnetLive` check and default to `mainnet`](https://radixdlt.atlassian.net/browse/ABW-2350)
[(Android) Upgrade to RET and GW v1.0.0](https://radixdlt.atlassian.net/browse/ABW-2312)

### Notes
* Mainnet is the new default
* Stokenet has become a known network which is selectable and not deletable according to this [conversation](https://rdxworks.slack.com/archives/C031A0V1A1W/p1695736523222039).
* Update to 1.0.0 GW and RET

### Testing
* Switch back and forth in stokenet and mainnet.
* Add your own GW
* if you already have a profile, that does not invalidate your profile. You need to delete the wallet in order to see the preselected GWs. This does not break compatibility
